runtime/src/system_instruction_processor.rs

```diff
@@ -231,7 +257,8 @@ mod tests {
             KeyedAccount::new(&to, false, &mut to_account),
         ];
         // fail to create an account with a sysvar id
-        let result = create_system_account(&mut keyed_accounts, 50, 2, &system_program::id());
+        let result =
+            create_system_account(&mut keyed_accounts, 50, 2, &system_program::id(), false);
```

Comment:
instead of touching every call to create_system_account(), how about introducing a new function?  create_rent_exempt_account()?

Reply:
nevermind.  this is on the instruction processor side, thought it was on the instruction generator side